code/lib/core-common/package.json

```diff
@@ -77,11 +77,6 @@
     "type-fest": "^2.19.0",
     "typescript": "~4.9.3"
   },
-  "peerDependenciesMeta": {
-    "typescript": {
```

Comment:
Why is typescript in `devDependencies` and not `dependencies`?  Does that mean it's being bundled into `@storybook/core-common`?

Reply:
> Does that mean it's being bundled into @storybook/core-common?

No, not necessarily, only if the bundler (tsup) would be able to find a dependency chain from any entrypoint to the `typescript ` module. Which if that happened I would find very strange because why would core-common import typescript as a runtime dependency anywhere?

Reply:
Right, okay, I keep forgetting that part of things, thanks for being patient with me.

Reply:
> Why is typescript in devDependencies and not dependencies?

Because if it defined it as a regular dependency, it would also add that module as a dependency for users to install, which I don not think is something we want, right?